CHANGELOG.md

```diff
@@ -14,6 +14,11 @@
 #### New Features
 
 #### _**Note**_: Breaking changes in the API
+* Refactoring #3654: 
+    * Removed deprecated KubernetesClient.customResource / RawCustomResourceOperationsImpl, please use the generic resource api instead 
+    * Removed deprecatedHttpClientUtils.createHttpClient(final Config config, final Consumer<OkHttpClient.Builder> additionalConfig), please use the OkHttpClientFactory instead
+    * Removed deprecated methods on SharedInformerFactory dealing with the OperationContext
+* Refactoring #2838: Readiness/OpenShiftReadiness moved from client.internal.readiness to client.readiness
```

Comment:
Some of these should definitely go into the https://github.com/fabric8io/kubernetes-client/blob/master/doc/MIGRATION-v6.md

Reply:
Since this list is bound to grow off limits, maybe it's better to just link to the migration guide WDYT?